src/main/java/Duke.java

```diff
@@ -1,10 +1,94 @@
+
+import java.util.Scanner;
+
 public class Duke {
+    public static void printUnderBar(){
+        System.out.println("\t____________________________________________________________");
+    }
+    public static void printFormAbove(){
+        printUnderBar();
+        System.out.println("\t Got it. I've added this task:");
+    }
+    public static void printCnt(int cnt){
+        System.out.println("\t Now you have " + cnt + " tasks in the list.");
+        printUnderBar();
+    }
+    public static void printMark(){
+        printUnderBar();
+        System.out.println("\t Nice! I've marked this task as done:");
+    }
+    public static void printUnMark(){
+        printUnderBar();
+        System.out.println("\t OK, I've marked this task as not done yet:");
+    }
+    public static void printList(){
+        printUnderBar();
+        System.out.println("\t Here are the tasks in your list:");
+    }
```

Comment:
Lots of magic variables. Maybe you could consider having a
final static String CONSTANT = "Some string";